front/app/components/ConfigurationMap/components/MapHelperOptions.tsx

```diff
@@ -59,7 +59,7 @@ const MapHelperOptions = ({ mapConfig, mapView }: Props) => {
           coordinates: [projectedPoint.longitude, projectedPoint.latitude],
         },
 
-        zoom_level: mapView.zoom.toString(),
+        zoom_level: getMapZoom(mapView.zoom, mapView.scale),
       });
```

Comment:
Added a new util to get the map zoom from either the MapView.Zoom, or MapView.Scale (some map types like the Web Map Lahti is using don't use the "Zoom" property).